src/components/Form.js

```diff
@@ -51,57 +53,58 @@ export default function Form({ handleData }) {
           {...register('foodName', {
             required: {
               value: true,
-              message: 'Wie willst Du etwas wiederfinden ohne Namen?',
+              message:
+                'How do you want to find something again without a name?',
             },
             maxLength: {
               value: 20,
-              message: 'ups, Limit von 2o Buchstaben erreicht',
+              message: 'ups, limit of 2o letters reached',
```

Comment:
```suggestion
              message: 'ups, limit of 20 characters reached',
```

Reply:
many thanks, you are right, i changed this